docs/release-notes/1200-fix-prior-batches-processed-backfilling.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Bug Fixes**
+
+- Fix a bug with a database migration in v0.12.13 which caused it to run slow and backfill
```

Comment:
The buggy database migration is in `0.13.0`, not `0.12.13`, no?

Reply:
Yep.